torch/distributed/rendezvous.py

```diff
@@ -1,12 +1,15 @@
 try:
     from urllib.parse import urlparse, urlunparse
 except ImportError:
-    from urlparse import urlparse, urlunparse
+    # Disable typechecking for Python2 modules
+    from urlparse import urlparse, urlunparse  # type: ignore
```

Comment:
should we remove it since we no longer support python2
```suggestion
    raise ImportError("urllib cannot be found, urlparse from python2 is no longer supported")
```

Reply:
Good point. Fixed as suggested.